src/query/functions-v2/src/scalars/control.rs

```diff
@@ -140,4 +150,58 @@ pub fn register(registry: &mut FunctionRegistry) {
             }),
         }))
     });
+    registry.register_1_arg_core::<NullableType<GenericType<0>>, BooleanType, _, _>(
+        IS_NULL,
+        FunctionProperty::default(),
+        |_| None,
+        |arg, _| match &arg {
+            ValueRef::Column(NullableColumn { validity, .. }) => {
+                let bitmap = Bitmap::from_iter(validity.iter().map(|v| !v));
+                Ok(Value::Column(bitmap))
+            }
+            ValueRef::Scalar(None) => Ok(Value::Scalar(true)),
+            ValueRef::Scalar(Some(_)) => Ok(Value::Scalar(false)),
+        },
+    );
+    registry.register_1_arg_core::<NullType, BooleanType, _, _>(
+        IS_NULL,
+        FunctionProperty::default(),
+        |_| None,
```

Comment:
```suggestion
        |_| Some(NullableDomain {has_null:true, value: None}),
```

Reply:
I think should be:
```rust
Some(BooleanDomain { has_true: true, has_false: false,})
```
Because return type is BooleanType

Reply:
Oh yes lol